src/core/qfieldcloudprojectsmodel.h

```diff
@@ -356,6 +422,9 @@ class QFieldCloudProjectsModel : public QAbstractListModel
         QString lastLocalExportedAt;
         QString lastLocalExportId;
         QString lastLocalPushDeltas;
+
+        QDateTime lastRefreshDt;
```

Comment:
What's Dt for

Reply:
datetime

Reply:
Hmm... That's at least not an existing common practice

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.date.html
or
```
import datatime as dt
```

Quite common in Python, but would change it.

Reply:
renamed to `lastRefreshDt` to `lastRefreshedAt`